docs/add-ons/index.md

```diff
@@ -49,40 +49,42 @@ In order to deploy an add-on with default configuration, simply enable the add-o
 
 ```hcl
 module "eks-blueprints-kubernetes-addons" {
```

Comment:
we can change the module names to use underscore (`_`) instead of hyphen (`-`) as a best practise and be consistent with TF standards. This might need changing in many other places as well